Release.toml

```diff
@@ -59,3 +59,7 @@ version = "1.1.4"
     "migrate_v1.1.3_kubelet-cpu-manager.lz4",
 ]
 "(1.1.3, 1.1.4)" = []
+"(1.1.4, 1.1.5)" = [
```

Comment:
Shouldn't this be 1.2.0? Given that this is a new feature that we will introduce in the next release?